server/serializers/subscriptions.go

```diff
@@ -71,6 +73,8 @@ type SubscriptionDetails struct {
 	ChannelType      string `json:"channelType"`
 	SubscriptionID   string `json:"subscriptionID"`
 	CreatedBy        string `json:"createdBy"`
+	TargetBranch     string `json:"targetBranch"`
+	Repository       string `json:"repository"`
```

Comment:
are these fields optional? if not we should validate them in a function used for validating payload `IsSubscriptionRequestPayloadValid`

Reply:
yes these are optional